client/multi-currency/index.js

```diff
@@ -127,13 +129,68 @@ if ( storeSettingsSection ) {
 	)[ 0 ];
 	storeSettingsContentContainer.appendChild( storeSettingsContent );
 
+	// Place the Store Settings element inside of storeSettingsContainer
+	const storeSettingsNote = document.querySelector(
+		'#wcpay_currencies_settings_section > div > p'
+	);
+
+	if ( storeSettingsNote )
+		storeSettingsContentContainer.appendChild( storeSettingsNote );
+
 	// Position and style Submit button
 	submitButton.style.textAlign = 'right';
 	submitButton.style.display = 'flex';
 	submitButton.classList.add( 'wcpay-settings-layout' );
 	submitButton.classList.remove( 'submit' );
 
 	toggleSettingsSectionDisplay();
+
+	const previewPanePlaceholder = document.querySelector(
+		'#preview-pane-placeholder'
+	);
+
+	const getFirstEnabledCurrencyFromList = () => {
+		const currenciesList = enabledCurrenciesListItemsExceptPlaceholders();
+		if ( 1 < currenciesList.length ) {
+			return currenciesList
+				.slice( 1, 2 )[ 0 ]
+				.classList.toString()
+				.replace( 'enabled-currency ', '' )
+				.trim()
+				.toUpperCase();
+		}
+		return null;
+	};
+
+	const PreviewModalHelper = () => {
+		const [ isPreviewModalOpen, setPreviewModalOpen ] = useState( false );
+
+		/* eslint-disable jsx-a11y/anchor-is-valid */
+		return (
+			<>
+				<a
+					href="#"
+					onClick={ () => {
+						setPreviewModalOpen( true );
+					} }
+				>
+					{ __( 'Preview', 'woocommerce-payments' ) }
+				</a>
+				<PreviewModal
+					isPreviewModalOpen={ isPreviewModalOpen }
+					setPreviewModalOpen={ setPreviewModalOpen }
+					isAutomaticSwitchEnabledValue={ true }
+					isStorefrontSwitcherEnabledValue={ false }
```

Comment:
I am curious why don't we need this switcher in preview too?

Reply:
![image](https://user-images.githubusercontent.com/3295/134417564-e7b35689-7574-4d51-9c90-057eced6897d.png)

The *Preview* link is bound to the auto currency switch feature, and I thought that it will only provide the preview for that feature only. In the onboarding, the link was near the `Continue` button and it provided the preview for both features.

Reply:
Gotcha, thanks.